File: Python/mario/mario.py
from cs50 import get_int

while True:
    # Prompting the user for the Height
    height = get_int("Height: ")
    if height > 0 and height < 9:
        break
for i in range(height):
    # Printing the blocks
    print(" " * (height - (i + 1)), end="")
    print("#" * (i + 1), end="  ")
    print("#" * (i + 1))

File: C/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt user for the Height
    int n;
    do
    {
        n = get_int("The Height: ");
    }
    while (n > 8 || n < 1);
    // Building the Spaces
    int z = n;
    for (int i = 0 ; i < n ; i++)
    {
        int m = 1;
        while (m < z)
        {
            printf(" ");
            m++;
        }
        z--;
        // Building the left-aligned Pyramid
        for (int j = 0; j <= i; j++)
        {
            printf("#");
        }
        printf("  ");
        // Building the -aligned Pyramid
        for (int x = 0; x <= i; x++)
        {
            printf("#");
        }
        printf("\n");
    }
}